### ISSUE TITLE
FUTO

### ISSUE THREAD

**Issue by ssddanbrown** [ssddanbrown/Open-Source-Confusion-Cases#49]
Company that has open source listed as one of their five pillars.
Released their app "GrayJay" via a video by Louis Rossmann, who described it as open source, with restrictions to prevent malware/adware, while under a more restrictive license. Details here:
https://hiphish.github.io/blog/2023/10/18/grayjay-is-not-open-source/

Founder touched on this in video (14:11) here:
https://www.youtube.com/watch?v=k3eycjekIAk&t=851s
But shows misunderstanding of what the concerns were in regard to open source.

Attempted to define their own open source definition, which I've detailed and pleaded against on my blog:
https://danb.me/blog/futo-open-source-definition/
Posted in their Zulip chat awaiting official feedback.

**Comment by SethFalco** [comment]
There's been an update on this btw!

TL;DR: They will now refer to their none OSI approved licenses at "Source First" instead of "Open-Source".

Reference: https://peertube.futo.org/w/229204e8-af1b-4904-8f72-04cf9ac9cc2b

**Comment by ssddanbrown** [comment]
Yeah, Good to hear they listened to feedback!
I'll still add this since it's an interesting case, but as a positive resolved example. 

Some other relevant links for this case:

- https://www.reddit.com/r/selfhosted/comments/1da8nld/this_week_in_selfhosted_7_june_2024/
- https://www.youtube.com/watch?v=zaoXHT8BfV4
- https://www.youtube.com/watch?v=FSgDANlRLXE
